### PR TITLE
Add mix task to import user from csv

### LINKED ISSUE #155: Assign students, leader, mentor to groups from IVLE

### PR DESCRIPTION
**Pull request by indocomsoft**
Fixes #155

### CODE REVIEW

File: test/cadet/accounts/accounts_test.exs [hunk 1]
```diff
@@ -204,4 +204,33 @@ defmodule Cadet.AccountsTest do
...
+    test "existing user" do
+      user = insert(:user)
+      user_params = params_for(:user, nusnet_id: user.nusnet_id)
+      Accounts.insert_or_update_user(user_params)
+
+      updated_user =
+        User
+        |> where(nusnet_id: ^user.nusnet_id)
+        |> Repo.one()
+
```
Comment (tuesmiddt) [inline test/cadet/accounts/accounts_test.exs]: Prefer if you also `assert user.id == updated_user.id` to check that it's the same user.

File: test/cadet/course/course_test.exs [hunk 2]
```diff
@@ -169,4 +169,54 @@ defmodule Cadet.CourseTest do
       |> Enum.each(&assert(Repo.get(Material, &1.id) == nil))
     end
   end
+
+  describe "get_or_create_group" do
+    test "existing group" do
+      group = insert(:group)
+
+      {:ok, group_db} = Course.get_or_create_group(group.name)
+
```
Comment (tuesmiddt) [inline test/cadet/course/course_test.exs]: Same as for users, check id is same for both `group` and `group_db`.

File: priv/repo/migrations/20180815021708_remove_not_null_from_leader_in_group.exs [hunk 3]
```diff
@@ -0,0 +1,19 @@
+defmodule Cadet.Repo.Migrations.RemoveNotNullFromLeaderInGroup do
```
Comment (tuesmiddt) [inline priv/repo/migrations/20180815021708_remove_not_null_from_leader_in_group.exs]: As discussed, should also drop unique constraint on `leader_id` in `groups` table.

File: test/cadet/course/course_test.exs [hunk 4]
```diff
@@ -169,4 +169,54 @@ defmodule Cadet.CourseTest do
...
+    test "existing group" do
+      group = insert(:group)
+      group_params = params_with_assocs(:group, name: group.name)
+      Course.insert_or_update_group(group_params)
+
+      updated_group =
+        Group
+        |> where(name: ^group.name)
+        |> Repo.one()
+
```
Comment (tuesmiddt) [inline test/cadet/course/course_test.exs]: Ditto.

File: test/cadet/accounts/accounts_test.exs (hunk 1 above)
Comment (indocomsoft) [reply test/cadet/accounts/accounts_test.exs]: Done

**Pull request merged**